64jam.p8

```diff
@@ -271,26 +274,38 @@ function drawcopter(x,y)
 	if player.dir == 7 then 
 		spr(96,x,y,1,1,true,false)
 	end
-		if player.dir == 10 then 
+	if player.dir == 10 then 
 		spr(97,x,y,1,1,true,false)
 	end
 	
+
+
+
+	--
+	-- rotor drawing code
+	--
+
+	TI_ROTOR_TOP = 67  -- TI - Tile Index
```

Comment:
Very cool.  I'm wondering if I can clear those blank spaces in the sprite sheet (which slow the rotor speed down) by having a counter associated with it, and only updating the sprite when the counter%2 = true.  I'm keen to free up all the tiles we can get.